### DISCUSSION TITLE
How to find Beta Coefficients in Linear Regression model in PyCaret?

### DISCUSSION THREAD

**Discussion by larun1916** [pycaret/pycaret#3510] [Ideas]
Is there a way to find Beta coefficients for Linear Regression in PyCaret? These Beta Coefficients would help me find the effect of each of the independent variables on the y variable. I am unable to figure it out after a thorough search. Appreciate if someone could help me here.

**Comment by ngupta23** [comment]
The underlying model is a scikit estimator. You can probe that to get the coefficients.
